From 89c4be6393b0c0117e3564a2a39105407238406b Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Tue, 21 Dec 2021 07:42:43 +0200
Subject: [PATCH] [SW-67660] gaudi2_arc: preboot size check to 127KB

As per recent change in flash layout for ARC boot flow to accommodate
preboot size exceeding 64KB, this patch changes the size check in
preboot compilation to 127KB. (1KB as a guard)

Change-Id: Ie9ecc79d4414e0d134a987bb0eb5a8f3cfb38a4d
---

diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index cf98f06..2c95e38 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -2,7 +2,7 @@
 
 FW_SIZE := 0x8000
 PPBOOT_MAX_SIZE := 16384
-PREBOOT_MAX_SIZE := 64512
+PREBOOT_MAX_SIZE := 130048
 
 LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
 
